db/R/dbfiles.R

```diff
@@ -39,31 +39,31 @@ dbfile.input.insert <- function(in.path, in.prefix, siteid, startdate, enddate,
   
   # find mimetype, if it does not exist, it will create one
   mimetypeid <- get.id("mimetypes", "type_string", mimetype, con, create=TRUE)
-  
+ 
   # find appropriate format, create if it does not exist
```

Comment:
this scares me since this will be the exact opposite logic of what we had before.